pkg/controller/plan/kubevirt.go

```diff
@@ -2846,9 +2868,10 @@ func (r *KubeVirt) CreatePvcForNfs(pvcNamePrefix, pvName, vmID string) (pvcName
 	labels := map[string]string{"provider": r.Plan.Provider.Source.Name, "app": "forklift", "migration": string(r.Migration.UID), "plan": string(r.Plan.UID), "ova": OvaPVCLabel, kVM: vmID}
 	pvc := &core.PersistentVolumeClaim{
 		ObjectMeta: meta.ObjectMeta{
-			GenerateName: pvcNamePrefix,
-			Namespace:    r.Plan.Spec.TargetNamespace,
-			Labels:       labels,
+			GenerateName:    pvcNamePrefix,
+			Namespace:       r.Plan.Spec.TargetNamespace,
+			OwnerReferences: setOwnerReferences(r.Plan),
```

Comment:
AFAIK pvc is owned by the dv, can you check that we don't break the delete chain here ?

I think we should own the DV it should own the PVC and it should own the PV

Reply:
this PVC should own the PV, see comment about PV owner